SDStateTableView/Classes/SDStateTableView.swift

```diff
@@ -132,6 +132,7 @@ public class SDStateTableView: UITableView {
     var actionButton = UIButton.autolayoutView()
     
     var buttonAction: (() -> Void)?
+    private var buttonConfigCallback: (UIButton) -> Void
```

Comment:
I am not sure if you have faced this issue or not, seems a valid one, i am facing compilation error because of this newly introduced callback, is there a way we can make it optional?
```
    private var buttonConfigCallback: ((UIButton) -> Void)?
```
<img width="1348" alt="Screenshot 2020-05-17 at 2 34 01 AM" src="https://user-images.githubusercontent.com/14096115/82130225-214aa100-97e7-11ea-97b5-36f3930584b2.png">

Reply:
Fixed.

Reply:
Sorry, i did not look into the implementation carefully since i was working late night @boda-taljo . You ca using this `private var buttonConfigCallback: ((UIButton) -> Void)?` to preserve the state of the table. But i guess this `buttonConfigCallback: ((UIButton) -> Void)? ` is already available inside `public var currentState: SDStateTableViewState` , do we need that extra variable here?